user_guide_src/source/testing/feature.rst

```diff
@@ -198,6 +198,14 @@ Asserts that the Response is an instance of RedirectResponse.
 
     $result->assertRedirect();
 
+**assertRedirectedTo()**
```

Comment:
docs is not yet updated

Reply:
@lonnieezell Could you update and fix the conflict in changelog?